VRDR.Tests/DeathRecord_Should.cs

```diff
@@ -752,29 +752,29 @@ public void Get_INTERVAL1B()
             Assert.Equal("6 days", ((DeathRecord)XMLRecords[0]).INTERVAL1B);
         }
 
-        [Fact]
-        public void Set_CODE1B()
-        {
-            Dictionary<string, string> code = new Dictionary<string, string>();
-            code.Add("code", "code 2");
-            code.Add("system", "system 2");
-            code.Add("display", "display 2");
-            SetterDeathRecord.CODE1B = code;
-            Assert.Equal("code 2", SetterDeathRecord.CODE1B["code"]);
-            Assert.Equal("system 2", SetterDeathRecord.CODE1B["system"]);
-            Assert.Equal("display 2", SetterDeathRecord.CODE1B["display"]);
-        }
-
-        [Fact]
-        public void Get_CODE1B()
-        {
-            Assert.Equal("I21.9", ((DeathRecord)JSONRecords[0]).CODE1B["code"]);
-            Assert.Equal("http://hl7.org/fhir/sid/icd-10", ((DeathRecord)XMLRecords[0]).CODE1B["system"]);
-            Assert.Equal("Acute myocardial infarction, unspecified", ((DeathRecord)JSONRecords[0]).CODE1B["display"]);
-            Assert.Equal("I21.9", ((DeathRecord)XMLRecords[0]).CODE1B["code"]);
-            Assert.Equal("http://hl7.org/fhir/sid/icd-10", ((DeathRecord)JSONRecords[0]).CODE1B["system"]);
-            Assert.Equal("Acute myocardial infarction, unspecified", ((DeathRecord)XMLRecords[0]).CODE1B["display"]);
-        }
+        // [Fact]
+        // public void Set_CODE1B()
+        // {
+        //     Dictionary<string, string> code = new Dictionary<string, string>();
+        //     code.Add("code", "code 2");
+        //     code.Add("system", "system 2");
+        //     code.Add("display", "display 2");
+        //     SetterDeathRecord.CODE1B = code;
+        //     Assert.Equal("code 2", SetterDeathRecord.CODE1B["code"]);
+        //     Assert.Equal("system 2", SetterDeathRecord.CODE1B["system"]);
+        //     Assert.Equal("display 2", SetterDeathRecord.CODE1B["display"]);
+        // }
+
+        // [Fact]
+        // public void Get_CODE1B()
+        // {
+        //     Assert.Equal("I21.9", ((DeathRecord)JSONRecords[0]).CODE1B["code"]);
+        //     Assert.Equal("http://hl7.org/fhir/sid/icd-10", ((DeathRecord)XMLRecords[0]).CODE1B["system"]);
+        //     Assert.Equal("Acute myocardial infarction, unspecified", ((DeathRecord)JSONRecords[0]).CODE1B["display"]);
+        //     Assert.Equal("I21.9", ((DeathRecord)XMLRecords[0]).CODE1B["code"]);
+        //     Assert.Equal("http://hl7.org/fhir/sid/icd-10", ((DeathRecord)JSONRecords[0]).CODE1B["system"]);
+        //     Assert.Equal("Acute myocardial infarction, unspecified", ((DeathRecord)XMLRecords[0]).CODE1B["display"]);
+        // }
```

Comment:
delete?

Reply:
I'm leaving the CODE1X stuff pending Pete's review.